python/ShipGeo.py

```
@@ -0,0 +1,77 @@
+import shipunit as u
+
+fDesign = 2
+
+
+class ShipGeo:
+    def __init__(self, z=0):
+        self.z = z
+
+
+def zPositions():
+    main = sys.modules["__main__"]
```

Ruff (0.15.12)

[error] Undefined name `sys`

(F821)